networkx/algorithms/voronoi.py

```diff
@@ -25,7 +25,7 @@ def voronoi_cells(G, center_nodes, weight="weight"):
     G : NetworkX graph
 
     center_nodes : set
-        A nonempty set of nodes in the graph `G` that represent the
+        A nonempty set of nodes in the graph $G$ that represent the
```

Comment:
Technically `` `G` `` is correct according to the [numpydoc docstring standard](https://numpydoc.readthedocs.io/en/latest/format.html#parameters): function parameters are supposed to be enclosed in single backticks when referenced elsewhere in the docstring.

Reply:
Good to know! I will revert it

Reply:
It's done